### ISSUE TITLE
Help text doesn't appear in command --help

### ISSUE THREAD

**Issue by apnewberry** [pypa/pipenv#2381]
```
$ pipenv --version
pipenv, version 2018.05.18

$ pipenv check --help
Usage: pipenv check [OPTIONS] [ARGS]...

Options:
  --three / --two  Use Python 3/2 when creating virtualenv.
  --python TEXT    Specify which version of Python virtualenv should use.
  --system         Use system Python.
  --unused TEXT    Given a code path, show potentially unused dependencies.
  -h, --help       Show this message and exit.

```

There's a field called `short_help` ("Checks for security vulnerabilities and against PEP 508 markers provided in Pipfile.") which appears when running `pipenv` alone, but it doesn't appear when running `pipenv check --help`. It probably should appear there, because that's what people run when they want to know what `pipenv check` does.

See also: https://github.com/pypa/pipenv/pull/1155

**Comment by uranusjr** [comment]
Contributions are welcomed!

**Comment by toxinu** [comment]
In order to have `short_help` printed with `pipenv check` command we must declare help text as `command` function docstring. Is it something we want to do?

Side-effect is that help will be truncated by `click` like that:
```
Commands:
  check      Checks for security vulnerabilities and...
  clean      Uninstalls all packages not specified in Pipfile.lock.
```
Strange behavior that `click` doesn't show `sort_help` when running `pipenv check` as excepted...

**Comment by techalchemy** [comment]
We have a giant blob of text with this info in `core.py` which might be what you’re looking for

**Comment by erinxocon** [comment]
This has been fixed and approved in #2578
